CASMNET-1805 | network upgrades for CANU specific things main (#2259)
Cray-HPE/docs-csm: operations/network/management_network/generate_switch_configs.md

    * See [Install/Upgrade CANU](canu_install_update.md).
* Validated SHCD.
  * See [Validate SHCD](validate_shcd.md).
* JSON output from validated SHCD.
  * See [Validate SHCD](validate_shcd.md).
* System Layout Service (SLS) input file.
  * If generating CSM 1.2 configurations, the SLS file must be updated prior to generating configurations.
  * See [Collect Data](collect_data.md).
* Generate custom switch configuration.
  * See [CANU custom configuration](https://github.com/Cray-HPE/canu/blob/develop/docs/network_configuration_and_upgrade/custom_config.md).

## Generate configuration files

Ensure that the correct architecture (`-a` parameter) is selected for the setup in use.

The following are the different architectures that can be specified:

* `Tds` – Aruba-based Test and Development System. These are small systems characterized by Kubernetes NCNs cabled directly to the spine.
* `Full` – Aruba-based Leaf-Spine systems. These are usually customer production systems.
* `V1` – Any Dell and Mellanox-based systems.

Generating a configuration file can be done for a single switch, or for the full system. Below are example commands for both scenarios:

> **Important:** Modify the following items in your command:

Reviewer: ```suggestion
**Important:** Modify the following items in the command:
```

## Prerequisites

> * CANU installed with version 1.6.13 or greater.

Reviewer 2: ```suggestion
* CANU installed with 1.6.13 or later versions.
```

  * Run `canu --version` to see version.
  * If doing a CSM install or upgrade, a CANU RPM is located in the release tarball. For more information, refer to the [Update CANU From CSM Tarball](canu/update_canu_from_csm_tarball.md) procedure.

>   * Alternatively: Upgrade or install latest version of CANU from GitHub [Install/Upgrade CANU](canu_install_update.md)

Reviewer 2: ```suggestion
  * Alternatively, upgrade or install latest version of CANU from GitHub [Install/Upgrade CANU](canu_install_update.md).
```
